how-to-use-azureml/machine-learning-pipelines/intro-to-pipelines/aml-pipelines-schedule-with-pipeline-parameter-built-in-functions.ipynb

```diff
@@ -0,0 +1,392 @@
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Create a schedule with pipeline parameter expressions for published pipeline endpoint"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "pipeline_endpoint_schedule = Schedule.create_for_pipeline_endpoint(\n",
+    "    ws,\n",
+    "    name=\"MyPipelineEndpointSchedule\",\n",
+    "    pipeline_endpoint_id=pipeline_endpoint_id,\n",
+    "    experiment_name=\"Schedule-endpoint-pipeline-parameter-sample\",\n",
+    "    description=\"Create for pipeline endpoint with pipeline parameter expressions\",      \n",
+    "    pipeline_parameters=pipeline_parameters,\n",
+    "    recurrence=recurrence) "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Update a schedule with pipeline parameter expressions for published pipeline endpoint  \n",
+    "**Note**: If you want to update pipeline parameter expression of published pipeline endpoint, you need to update the previous schedule in below way, as simply republishing the pipeline will not work."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "update_pipeline_parameters = {\n",
+    "    \"dayOfMonth\": \"@dayOfMonth(utcNow())\",\n",
+    "    \"dayOfWeek\": \"@dayOfWeek(utcNow())\",\n",
+    "}\n",
+    "\n",
+    "pipeline_endpoint_schedule.update(\n",
+    "    name=\"MyUpdatedPipelineEndpointSchedule\",\n",
+    "    description=\"Based on time, update pipeline endpoint with parameter expressions\",\n",
+    "    pipeline_parameters=update_pipeline_parameters,\n",
+    "    recurrence=update_recurrence)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Disable the schedule "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "pipeline_endpoint_schedule.disable()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Frequently Asked Questions"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "[Q1](#1): Why are my input pipeline parameter expressions not resolved?  \n",
+    "[Q2](#2): What should I do when I meet error \"Invalid pipeline parameter expression\"?  \n",
+    "[Q3](#3): Why do new triggered runs still use previous pipeline parameter expressions although I have republished with new expressions?  \n",
+    "[Q4](#4): Why are some parameter expressions resolved while others are not?    "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "1. <span id = \"1\">**Q: Why are my input pipeline parameter expressions not resolved?**</span>  \n",
+    "    **A**: Please check if the expressions are in the expression list and use correct case, otherwise they are used as raw string."
```

Comment:
Pipeline parameter expressions are case sensitive. Please check if your input expression is in supported built-in functions, otherwise used as raw string.